test/images/regression-issue-74839/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine:3.6
```

Comment:
Can we avoid alpine and instead try to use [distroless](https://github.com/GoogleContainerTools/distroless)?

Reply:
Should be doable assuming we don't have special dependency here. Ref https://github.com/kubernetes/dns/pull/294 for a similar change.